xwalk.gyp

```diff
@@ -10,6 +10,9 @@
       }],
       ['OS=="android"', {
         'enable_extensions': 1,
+        'sdk_version': '<!(python ../build/util/version.py -f SDK_VERSION -t "@SDK@")',
+        'min_sdk_version': '<!(python ../build/util/version.py -f SDK_VERSION -t "@MIN_SDK@")',
+        'verify_xwalk_apk%': 0,
```

Comment:
It would be good if you could add a short comment to document this variable. See how Chromium's `build/common.gypi` has much more information than ours.